src/EventSourcingTests/SchemaChange/Upcasters.cs

```diff
@@ -734,14 +736,14 @@ private void Apply(ShoppingCartOpened @event)
     // Events in new namespace `New`
     namespace New
     {
-        public class ShoppingCart: AggregateBase
+        public class ShoppingCartAggregate: AggregateBase
```

Comment:
**QUESTION:** @Hawxy, is there any reason for renaming this class? I'd prefer not to have suffixes in the type names, especially those that are used in the public documentation.

Reply:
Good point, has been fixed.